upf/test/test_quantifier_remover.py

```diff
@@ -111,3 +111,19 @@ def test_robot_locations_visited(self):
             self.assertEqual(str(plan), str(uq_plan))
             new_plan = qr.rewrite_back_plan(uq_plan)
             self.assertEqual(str(plan), str(new_plan))
+
+    def test_timed_connected_locations(self):
+        problem = self.problems['timed_connected_locations'].problem
+        plan = self.problems['timed_connected_locations'].plan
+        qr = QuantifiersRemover(problem)
+        uq_problem = qr.get_rewritten_problem()
+        # Temporal problem still missing a __repr__ function
+        # self.assertIn("Exists", str(problem))
```

Comment:
Can't we add an `has_quantifiers` method to check this instead of relying on the repr?

Reply:
Could also be a viable option to use the` problem_kind EXISTENTIAL_PRECONDITIONS` , which option do you like better?

Reply:
Both are fine. You an implement a method `has quantifiers` by reading the problem kind

Reply:
Perfect! I am happy we had the same idea.